src/openrct2/network/ServerList.cpp

```diff
@@ -70,35 +70,40 @@ bool ServerListEntry::IsVersionValid() const
     return Version.empty() || Version == network_get_version();
 }
 
-std::optional<ServerListEntry> ServerListEntry::FromJson(const json_t* server)
+std::optional<ServerListEntry> ServerListEntry::FromJson(json_t& server)
 {
-    auto port = json_object_get(server, "port");
-    auto name = json_object_get(server, "name");
-    auto description = json_object_get(server, "description");
-    auto requiresPassword = json_object_get(server, "requiresPassword");
-    auto version = json_object_get(server, "version");
-    auto players = json_object_get(server, "players");
-    auto maxPlayers = json_object_get(server, "maxPlayers");
-    auto ip = json_object_get(server, "ip");
-    auto ip4 = json_object_get(ip, "v4");
-    auto addressIp = json_array_get(ip4, 0);
-
-    if (name == nullptr || version == nullptr)
+    auto port = Json::GetNumber<int32_t>(server["port"]);
+    auto name = Json::GetString(server["name"]);
+    auto description = Json::GetString(server["description"]);
+    auto requiresPassword = Json::GetBoolean(server["requiresPassword"]);
+    auto version = Json::GetString(server["version"]);
+    auto players = Json::GetNumber<uint8_t>(server["players"]);
+    auto maxPlayers = Json::GetNumber<uint8_t>(server["maxPlayers"]);
+    std::string ip = "";
```

Comment:
```suggestion
    std::string ip;
```
No need to initialize a `std::string` object